sql/core/src/main/scala/org/apache/spark/sql/streaming/DataStreamWriter.scala

```diff
@@ -198,8 +209,19 @@ final class DataStreamWriter[T] private[sql](ds: Dataset[T]) {
    *
    * You can set the following option(s):
    * <ul>
-   * <li>`timeZone` (default session local timezone): sets the string that indicates a timezone
-   * to be used to format timestamps in the JSON/CSV datasources or partition values.</li>
+   * <li>`timeZone` (default session local timezone): sets the string that indicates a time zone ID
```

Comment:
is the `timezone` option effective for writing? AFAIK it's for parsing only.

Reply:
If a pattern contains sub-pattern for time zone like `V` or `z`, which one formatter should output?

Reply:
ah got it

Reply:
shall we update the comment? It's not for parsing.

Reply:
Updated